moviebot/dialogue_manager/dialogue_state_tracker.py

```diff
@@ -323,13 +288,16 @@ def update_state_agent(self, agent_dacts):
                     self.ontology.user_requestable
                 )
 
-    def update_state_db(self, database_result=None, backup_results=None):
-        """Updates the state based on  the results fetched from the database.
+    def update_state_db(
+        self, database_result: List = None, backup_results: List = None
```

Comment:
Nit: List[Dict[str, Any]]

If possible I like to have at least one level in on lists and dictionaries.

Reply:
Done.